organize/emails.py

```diff
@@ -6,7 +6,7 @@
 
 def send_application_confirmation(event_application):
```

Comment:
So this file has this unused import which is causing flake 8 to fail `from django.utils.translation import gettext_lazy as _`. Could you kindly remove it? Thanks @marksweb

Reply:
Ah yes, of course. Too busy rushing things. 🤦‍♂️